src/main/java/com/blackducksoftware/integration/hub/alert/datasource/entity/DatabaseEntity.java

```diff
@@ -40,6 +40,10 @@
     @Column(name = "id")
     private Long id;
 
+    public DatabaseEntity(final Long id) {
+        this.id = id;
+    }
+
```

Comment:
Should we require that anything extending the DatabaseEntity class need to have an ID? What about the cases when we have a table that has a many to many relationship?

Reply:
We should probably rename DatabaseEntity to be more specific to configuration, but not in this pull request.